src/main/java/com/ait/lienzo/client/core/image/ImageElementProxy.java

```diff
@@ -0,0 +1,83 @@
+package com.ait.lienzo.client.core.image;
+
+import com.ait.lienzo.client.core.Context2D;
+import com.ait.lienzo.client.core.config.LienzoCore;
+import com.google.gwt.dom.client.ImageElement;
+import com.google.gwt.user.client.ui.Image;
+import com.google.gwt.user.client.ui.RootPanel;
+
+public class ImageElementProxy {
```

Comment:
Class formatted as Kie-Group class, not Lienzo's one. If we want to merge changes to upstream it will be an issue in the future.